webpack.config.js

```diff
@@ -0,0 +1,36 @@
+const defaultsDeep = require('lodash.defaultsdeep');
+const path = require('path');
+const webpack = require('webpack');
+
+
+const base = {
+    devtool: 'cheap-module-source-map',
+    module: {
+        rules: [{
+            test: /\.js$/,
+            loader: 'babel-loader',
+            include: path.resolve(__dirname, 'src')
+        }]
+    },
+    plugins: []
+        .concat(process.env.NODE_ENV === 'production' ? [
+            new webpack.optimize.UglifyJsPlugin({
+                include: /\.min\.js$/,
+                minimize: true
```

Comment:
Looking at it again, I don't think we need this — we won't be using minified js from this repo.